### PR TITLE
add urlparse() helper

### LINKED ISSUE #191: Add urlparser to check and alert
I have a string "http://foo.bar:123/blah" and I want to get the parsed url object.
This would be great to get:

In [4]: urllib3.util.parse_url("http://foo.bar:123/blah")
Out[4]: Url(scheme='http', auth=None, host='foo.bar', port=123, path='/blah', query=None, fragment=None)

@szuecs from controller

### PR DESCRIPTION
**Pull request by vetinari**
fixes #191

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: zmon_worker_monitor/zmon_worker/tasks/main.py [hunk 1]
```diff
@@ -634,6 +635,13 @@ def empty(v):
     return not bool(v)
 
 
+def urlparse(u):
```
Comment (mohabusama) [inline zmon_worker_monitor/zmon_worker/tasks/main.py]: this needs to go in ``build_default_context``. And I would rename ``u`` to ``url``

**Comment by vetinari** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**